src/Models/NonhydrostaticModels/update_nonhydrostatic_model_state.jl

```diff
@@ -24,10 +25,13 @@ function update_state!(model::NonhydrostaticModel, callbacks=[]; compute_tendenc
     end
 
     # Update all FieldTimeSeries used in the model
-    update_model_field_time_series!(model, model.clock)
+    #update_model_field_time_series!(model, model.clock)
```

Comment:
have to comment this back before merging

Reply:
Hopefully it works now (I was having issues with the recursion in it before)